### PR TITLE
Export to parquet: also export score

### PR DESCRIPTION
**Pull request by julien-c**
Follow-up to #151 to account for #152

### CODE REVIEW

File: src/routes/admin/export/+server.ts [hunk 1]
```diff
@@ -34,7 +34,14 @@ export async function POST({ request }) {
 		title: { type: "UTF8" },
 		created_at: { type: "TIMESTAMP_MILLIS" },
 		updated_at: { type: "TIMESTAMP_MILLIS" },
-		messages: { repeated: true, fields: { from: { type: "UTF8" }, content: { type: "UTF8" } } },
+		messages: {
+			repeated: true,
+			fields: {
+				from: { type: "UTF8" },
+				content: { type: "UTF8" },
+				score: { type: "INT_8", optional: true },
```
Comment (julien-c) [inline src/routes/admin/export/+server.ts:42]: @lhoestq @severo Out of curiosity, does our viewer support `INT_8` (and optional) data cells in Parquet files?

**Review by coyotte508 — APPROVED** [review]

File: src/routes/admin/export/+server.ts (hunk 1 above)
Comment (severo) [reply src/routes/admin/export/+server.ts:42]: do you have an example file to share?

Comment (lhoestq) [reply src/routes/admin/export/+server.ts:42]: also note that the viewer is not enabled on private datasets !

Comment (julien-c) [reply src/routes/admin/export/+server.ts:42]: yes i know!

**Pull request merged**